### ISSUE TITLE
Error in the Hooper method for conical diffuser/expansion

### ISSUE THREAD

**Issue by AllanBjerg** [CalebBell/fluids#60]
Dear Caleb,

It is a great tool that you have developed!

I am currently working with the Hooper method for a conical diffuser/expansion from the fluids.fittings library. I think there is a small error for the case of Re_1 < 4000 and angle < 45°. It returns K = K_sharp while it should return K = K_sharp*2.6*sin(angle/2).
 
Code is stated below:
    elif method == 'Hooper':
        if Re is None:
            raise ValueError("Method `Hooper` requires Reynolds number")
        if Re < 4000.0:
            return 2.0*(1.0 - beta*beta*beta*beta) # Not the same formula as Rennels
        if fd is None:
            fd = Clamond(Re=Re, eD=roughness/Di1)
        x = 1.0 - beta*beta
        K = (1.0 + 0.8*fd)*x*x
        if angle_rad > 0.25*pi:
            return K
        return K*2.6*sin(0.5*angle_rad)

Best regards,
Allan

**Comment by CalebBell** [comment]
Hi Allan,
I took a quick look and here is the original formula from the paper:

![image](https://github.com/CalebBell/fluids/assets/16520296/ce2aa310-e51c-4a1d-98c4-5fae72a0775d)

These look to me to be the formulas I've implemented. 
Can you clarify what you are asking? I don't understand.

**Comment by AllanBjerg** [comment]
Hi Caleb,

I agree that these formulas are implemented correct but I am talking about the diffuser and not the reduction.
![Expansion_Hooper](https://github.com/CalebBell/fluids/assets/135201877/49860706-81b7-4d49-9820-434b581be84e)

When I run the code for an expansion the result for K for a conical expansion at Re < 4000 and angle < 45° is the same as for a sharp expansion. As per the Hooper paper for these conditions the expression for K should be multiplied by: 2.6 x sin(angle/2). I don't think this happens in the code. See example of calculation below:
![Expansion_Hooper_fluids_example](https://github.com/CalebBell/fluids/assets/135201877/cf58fc74-901c-4768-911b-f5e1336e8aba)
Since we are in the range Re < 4000 and angle < 45° for the conical expansion, the K value should not be the same as for the sharp expansion. If the value is multiplied with 2.6 x sin(angle/2) the resulting K = 1.2617.

I hope that this clarifies my question.

Best regards,
Allan

**Comment by CalebBell** [comment]
I gave this issue another review. 
The 0.28 coefficient must have been a typo, I have corrected it to 0.18 in friction_plate_Martin_VDI. Thank you Andreas for the discovery. 
friction_plate_Martin_1999 is intended to implement the 1999 publication cited of the equation for friction factor. I have double checked that equations A, B, and C in the appendix of https://publikationen.bibliothek.kit.edu/1000034866/2579146 which is the 1999 Martin method, are those currently implemented. 
Pierre, Andreas: Thank you very much for the corrections. I am resolving the issue.
Caleb
